code/restful-api/restful-api/Models/Promocao.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+namespace RestfulApi.Models
+{
+    public class Promocao
```

Comment:
I think this should probably be linked with `Evento` instead of `Organizador`.

Why did you decide to go with `Organizador` instead?

Reply:
Removido o Organizador e criado o Eventos

Reply:
🆗